docs/using.rst

```diff
@@ -54,12 +54,18 @@ standalone_ Y    N    | Uses a "standalone" webserver to obtain a certificate.
                       | Requires port 80 or 443 to be available. This is useful on    tls-sni-01_ (443)
                       | systems with no webserver, or when direct integration with
                       | the local webserver is not supported or not desired.
+|dns_plugs| Y    N    | This category of plugins automates obtaining a certificate by dns-01_ (53)
+                      | modifying DNS records to prove you have control over a
+                      | domain. Doing domain validation in this way is
+                      | necessary to obtain certificates from Let's Encrypt.
```

Comment:
necessary to obtain wildcard certificates I assume

Reply:
but maybe you actually mean "...is the only way to obtain wildcard certificates..."

Reply:
Thanks for noticing the missing word "wildcard".

What's the difference between "is necessary to obtain wildcard certificates" and "is the only way to obtain wildcard certificates"?

Reply:
more down to earth real people talk

Reply:
also only vs. necessary aren't exactly the same

Reply:
I don't care that much though

Reply:
http://www.reactiongifs.com/wp-content/uploads/2013/11/real.gif